src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form 
+      action="#" 
+      method="post" 
+      name="personaleDate"
+    >
+      <fieldset class="block">
+        <legend class="information">Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form-field">
+            Name:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form-field">
+            How old are You?
+          <input 
+            type="number" 
+            name="old" 
+            min="0" 
+            max="100" 
+            value="12">
+        </label>
+
+        <label class="form-field">
+            Full date of birth:
+          <input 
+            type="date" 
+            name="data" 
+            placeholder="mm/dd/yyyy">
+        </label>
+
+        <label class="form-field lastForm">
+            I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="accept" 
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend class="registration">
+        Registration
+        </legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+          type="email" 
+          name="email" 
+          placeholder="email@example.com"
+          >
```

Comment:
```suggestion
          <input 
            type="email" 
            name="email" 
            placeholder="email@example.com"
          >
```
Потрібно звертати увагу на відступи.
Почитати можна тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation